components/expandable/w-expandable.vue

```diff
@@ -13,69 +13,49 @@ const props = defineProps({
   title: String,
   box: Boolean,
   bleed: Boolean,
-  info: Boolean,
   buttonClass: String,
   contentClass: String,
   chevron: { type: Boolean, default: true },
   as: { type: String, default: 'div' },
   animated: Boolean,
   ...modelProps({ modelDefault: absentProp }),
 });
+
 const emit = defineEmits(['expand', 'collapse']);
 const slots = useSlots();
 
 const expanded = props.modelValue === absentProp ? ref(false) : createModel({ props, emit });
 const contentComponent = computed(() => (props.animated ? expandTransition : 'div'));
-const showChevronUp = ref(expanded.value);
-// wExpandTransition emits its own events and we just bubble them, but for a normal DOM element we need to create them
+
 if (!props.animated) {
   watch(expanded, async (isExpanded) => {
     await nextTick();
     emit(isExpanded ? 'expand' : 'collapse');
+    expanded.value = isExpanded;
```

Comment:
Won't this just set `expanded` to whatever it already is? 🤔😅

Reply:
yeah you're right! We don't need this! 😄